lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -582,8 +582,12 @@ def _set_security_conf(self, data: PeerClusterRelData) -> None:
 
         secrets.put_object(Scope.APP, CertType.APP_ADMIN.val, data.credentials.admin_tls)
 
+        if not self.charm.tls.all_tls_resources_stored():
+            self.charm.tls.request_new_unit_certificates()
```

Comment:
is this needed outside of the CA / Admin cert rotation workflow in the main orchestrator? 
I was under the impression that the TLS workflow is independent from this and would have started without being explicitly requested in case of a new large deployment.

Reply:
This fixed an integration test we have for plugin-management in large deployments. As I did not test it for large deployments myself yet (this will be the scope of the next task for this pulse), I didn't see if working outside of the test.

It fixed the following case: Followers get the new CA from the main orchestrator via `relation-changed`, but didn't receive their new certs yet. I could not yet investigate why they didn't receive/update the certs, but requesting new ones helped here.

This could be subject of change when I work on the large-deployment-task.

Reply:
In general - I wouldn't worry much about the plugin management module, as it requires rework. 

Let's please remember this one. To clarify - I'm referring to the "happy path" when:
- +2 applications bootstrapped and related to the self-signed-certificates 
- both had their certificates issued successfully - with the exception of the app-admin certificate on the "non main orchestrator cluster" 
- with this logic, new units certificates will be reissued _again_

Reply:
Yes I understand what you mean, this should be avoided. I will address that in the PR for CA rotation in large deployments.

Reply:
has now been removed.